airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/GcsDestination.java

```diff
@@ -52,10 +56,18 @@ public AirbyteConnectionStatus check(final JsonNode config) {
       S3Destination.testMultipartUpload(s3Client, destinationConfig.getBucketName(), destinationConfig.getBucketPath());
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
+    } catch (final AmazonS3Exception e) {
+      LOGGER.error("Exception attempting to access the Gcs bucket: {}", e.getMessage());
```

Comment:
Maybe `AWS` not `Gcs`
Also, please look for other capitalization errors.

Reply:
Thank you for your comments. Changed it.